ssl/statem/extensions_clnt.c

```diff
@@ -579,7 +668,7 @@ EXT_RETURN tls_construct_ctos_psk_kex_modes(SSL *s, WPACKET *pkt,
     s->ext.psk_kex_mode = TLSEXT_KEX_MODE_FLAG_KE_DHE;
     if (nodhe)
         s->ext.psk_kex_mode |= TLSEXT_KEX_MODE_FLAG_KE;
-#endif
+#endif  /* OPENSSL_NO_TLS1_3 */
 
     return EXT_RETURN_SENT;
```

Comment:
Looks like there is a bug here in the existing code. `EXT_RETURN_SENT` should only be returned if the extension has actually been added. In the case of `OPENSSL_NO_TLS1_3` it isn't sent so we should return `EXT_RETURN_NOT_SENT`.